### PR TITLE
added-fix-to-my-nosetests-error-and-edited-server.py-to-increase-coverage

### PR DESCRIPTION
**Pull request by rohithreddy1**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -268,14 +268,15 @@ def initialize_mysql(test=False):
             conn = engine.connect()
         else:
             conn = connect_mysql('root', '', '127.0.0.1', 3306, 'nyudevops')
-
+            
+debug = (os.getenv('DEBUG', 'False') == 'True')
+port = os.getenv('PORT', '5000')
```
Comment (sadam0930) [inline server.py]: @rohithreddy1, are these supposed to be class variables? If so, can you move them to the top. It looks like they are part of the initialize_mysql method here.

Comment (rohithreddy1) [reply server.py]: @sadam0930 I think these can be global variables. I will move them to top.

Comment (0x932b) [reply server.py]: I don't think this is a good idea from a coding prospective to globalize these variables.
For the sake of coverage
using the coverage config files
https://stackoverflow.com/questions/5850268/how-to-test-or-mock-if-name-main-contents
should be better?

**Review by 0x932b — COMMENTED** [review]
I don't think this is a good idea from a coding prospective to globalize these variables.
For the sake of coverage
using the coverage config files
https://stackoverflow.com/questions/5850268/how-to-test-or-mock-if-name-main-contents
should be better?

File: Vagrantfile [hunk 2]
```diff
@@ -9,7 +9,9 @@ Vagrant.configure(2) do |config|
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://atlas.hashicorp.com/search.
   config.vm.box = "ubuntu/trusty64"
-
+  config.vm.synced_folder "./", "/vagrant",
```
Comment (0x932b) [inline Vagrantfile:12]: how does this solves the problem?
I am confused.
isn't this like the default setup?

File: server.py (hunk 1 above)
Comment (rohithreddy1) [reply server.py]: @u932b I agree with you. I will try to main from the coverage report

File: Vagrantfile (hunk 2 above)
Comment (rohithreddy1) [reply Vagrantfile:12]: @u932b There is some bug in windows which is causing this error. I am not sure how this solves my problem. I worked with professor and figured out that adding these lines in Vagrantfile would fix the bug. It did solve my problem.

**Review by 0x932b — APPROVED** [review]

**Pull request merged**